package/kernel/linux/modules/netfilter.mk

```diff
@@ -1048,6 +1048,40 @@ endef
 $(eval $(call KernelPackage,nfnetlink-queue))
 
 
+define KernelPackage/nfnetlink-cthelper
+  TITLE:=Netfilter User space conntrack helpers
+  FILES:=$(LINUX_DIR)/net/netfilter/nfnetlink_cthelper.ko
+  KCONFIG:=CONFIG_NF_CT_NETLINK_HELPER
+  AUTOLOAD:=$(call AutoProbe,nfnetlink_cthelper)
+  $(call AddDepends/nfnetlink,+kmod-nfnetlink-queue +kmod-nf-conntrack-netlink)
+endef
+
+define KernelPackage/nfnetlink-cthelper/description
+ Kernel modules support for a netlink-based connection tracking
+ userspace helpers interface
+endef
+
+$(eval $(call KernelPackage,nfnetlink-cthelper))
+
+
+define KernelPackage/nfnetlink-cttimeout
+  TITLE:=Netfilter conntrack expectation timeout
+  FILES:=$(LINUX_DIR)/net/netfilter/nfnetlink_cttimeout.ko
+  KCONFIG:=\
+    CONFIG_NF_CONNTRACK_TIMEOUT=y \
```

Comment:
This option compiles some extra code into `nf_conntrack.ko`. See https://github.com/torvalds/linux/blob/master/net/netfilter/Makefile#L10

Please post how much the size of this module increases.

Reply:
Good point. On 23.05, 150936 byes, with this 158744 - ~8k.

Happy to remove this from the PR, alternatively I think we can enable this only for selected targets where memory isn't a concern? Or maybe leave the option there but disable it by default? (Not sure how to do that though)

Reply:
Yes please remove it here.

Is this needed for your use case? 

If you need it you can add it to `config/Config-kernel.in` and activate it for not small flash devices.

`kmod-nf-conntrack-netlink` activates already some other options.

Reply:
I think this is what you meant -- I did a compile test and it appears for my device (E8450). Did I misunderstand you? This does mean that for small flash devices the package would be empty. Is that OK?

The timeout extension is helpful for some stateful applications that slowly stream data (e.g Sonos seems quite notorious for this) but so far I've been working on the Steam broadcast and it doesn't require this. So there are some usecases where it will work, and others that don't work so well.

Reply:
hey @hauke just a gentle nudge, if there's anything you'd like changed please let me know.

Happy holidays!